arch/arm/core/Kconfig

```diff
@@ -119,6 +140,9 @@ config ARM_NSC_REGION_BASE_ADDRESS
 	  the size and the alignment of the starting address for
 	  a Non-Secure Callable section, depending on the available
 	  security attribution unit (SAU or IDAU) for a given SOC.
+	  - This is ignored on the following SOCs, since their
```

Comment:
Well, I don't see clearing the reason for this note; maybe just makes this even more confusing than it is already :)

Reply:
Removed. The new ifdefs in linker.ld means this isn't needed.